ansible/roles/software/ckan/inventory/tasks/datapusher-classic.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: Remove old datapusher code
+  file: path="{{ datapusher_virtual_env }}" state=absent
+
+- include_tasks: build-pkgs.yml
+  vars:
+    build_pkg_name: "{{ datapusher_build_pkg_name }}"
+    build_pkg_repo: "{{ datapusher_build_pkg_repo }}"
+    build_pkg_requirements: "{{ datapusher_build_pkg_requirements }}"
+    virtualenv: "{{ datapusher_virtual_env }}"
+
+- name: "Datapusher json content-type temp fix (content-type: application/json does not work now)"
```

Comment:
FYI, once inventory-next ships, we'll be able to close https://github.com/GSA/datagov-deploy/issues/1177.
```suggestion
# TODO remove this code to resolve https://github.com/GSA/datagov-deploy/issues/1177
- name: "Datapusher json content-type temp fix (content-type: application/json does not work now)"
```